lib/users.rb

```diff
@@ -0,0 +1,39 @@
+require 'bcrypt'
+require_relative './database_connection'
+
+class User
+  def self.create(email:, password:)
+    encrypted_password = BCrypt::Password.create(password)
+    result = DatabaseConnection.query(
+      "INSERT INTO users (email, password) VALUES ($1, $2) RETURNING id, email;",
+      [email, encrypted_password]
+    )
+    User.new(id: result[0]['id'], email: result[0]['email'])
+  end
```

Comment:
Maybe need a guard condition to prevent people using an already-used email to sign up (currently can make 2 accounts using the same email which leaves you signed in, although when using login function, only the first one will let you login), or can leave as a nice-to-have thing to do later on